internal/controller/utils_test.go

```diff
@@ -65,6 +66,10 @@ func TestReconcileStatefulSet(t *testing.T) {
 		Spec: ecv1alpha1.EtcdClusterSpec{
 			Size:    3,
 			Version: "3.5.17",
+			// EtcdOptions: []string{
+			// 	"--max-wals=7",
+			// 	"--discovery-failbox=proxy",
+			// },
```

Comment:
Left commented out for the moment while validating that TestReconcileStatefulSet is running properly